octobot_services/interfaces/abstract_interface.py

```diff
@@ -55,3 +65,12 @@ def is_bot_ready():
     @abc.abstractmethod
     async def stop(self):
         raise NotImplementedError(f"stop is not implemented for {self.get_name()}")
+
+    @staticmethod
+    async def _create_user_commands_channel_if_not_existing() -> channels.Channel:
+        if AbstractInterface.user_commands_channel_producer is None:
+            channel = await channel_creator.create_channel_instance(service_channels.UserCommandsChannel,
+                                                                    channels.set_chan)
+            AbstractInterface.user_commands_channel_producer = service_channels.UserCommandsChannelProducer(channel)
```

Comment:
Why not using internal producer instead of creating a new static producer here ?